patches/runtime/0030-Add-TRUE-and-FALSE-definitions.patch

```diff
@@ -0,0 +1,191 @@
+From 15f59106c7f6cb99945d55e9ac280565e28e22bc Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Wed, 17 Mar 2021 01:32:33 -0500
+Subject: [PATCH 29/29] Add missing TRUE and FALSE definitions.
```

Comment:
What is the runtime repo doing about this?  Are they encountering the same issue?  I'm assuming this is a short-lived patch and will be removed in the next servicing release?

Reply:
runtime appears to use a specific OSX image.  I've logged https://github.com/dotnet/source-build/issues/2100 for us to look into this, that seems like it would really help us out.

Reply:
And yes, this is intended to be removed as soon as the lab machines are updated to Big Sur (or we switch to a frozen image).